packages/flutter/lib/src/cupertino/icon_theme_data.dart

```diff
@@ -18,8 +18,9 @@ class CupertinoIconThemeData extends IconThemeData with Diagnosticable {
   const CupertinoIconThemeData({
```

Comment:
I don't know that we need to add this to the CupertinoIconThemeData, from what I can tell, the splash effect is unique to Material.